### ISSUE TITLE
Error: 'calculate_mapping' is not an exported object from 'namespace:dyneval'

### ISSUE THREAD

**Issue by QidiFeng** [dynverse/dyneval#55]
Hi, dyneval group,

I failed to use calculate_mapping under dyneval package.
> dyneval::calculate_mapping
Error: 'calculate_mapping' is not an exported object from 'namespace:dyneval'

I did load dyneval library, and "?calculate_mapping" did help to show the documentation. But it shows no such function every time I used it.

Functions with the same problem including 'calculate_mapping_branches', 'calculate_metrics', and 'calculate_position_predict'.

Could you please help to find the problem? Thank you very much!

Best,
Qidi

**Comment by rcannood** [comment]
Hi Qidi! 

I think in most cases, it's recommended to use `calculate_metrics()` instead (which is an interface for all of the metrics). The function was not exported, but you could have accessed it with `dyneval:::calculate_mapping()`. In the @devel branch of dyneval, I now exported the functions you mentioned. Can you confirm that the devel branch solves your problem?

Robrecht

**Comment by QidiFeng** [comment]
Hi Robrecht,

Thank you very much for your reply. I think calculate_metrics() did solve my problem. 
But I don't know how to use the devel branch of dyneval. I failed to find a documentation of devel. 

Besides, when I tried to use metric_id 'him', it reported that 'Error in loadNamespace(name) : there is no package called ‘netdist’'. And I searched package 'netdist' by google, but still don't know how to install it. So how do I suppose to solve this problem?

Thanks again for your help!

Best,
Qidi
